### ISSUE TITLE
Không gõ được tiếng việt trong photoshop trên macOS

### ISSUE THREAD

**Issue by thuyld** [tuyenvm/OpenKey#113]
Mình vừa update bản mới nhất, thì vẫn bị lỗi không gõ được tiếng việt trong photoshop ( MacOS), mình bị dính từ bản 1.6 rồi.

P/s: thêm nữa là trên macOS khi update về thì mất luôn file openKey, mình phải tải lại bản mới trên github.

Bạn thử check xem có đúng ko?

**Comment by tuyenvm** [comment]
Mất file openkey là mất OpenKey trong Application ấy hả bạn?

**Comment by tuyenvm** [comment]
Còn phốthop bị từ 1.6 tức là phiên bản cũ hơn vẫn gõ tv được hr bạn?

**Comment by thuyld** [comment]
> Mất file openkey là mất OpenKey trong Application ấy hả bạn?

Uh, mất file trong Application luôn

**Comment by thuyld** [comment]
> Còn phốthop bị từ 1.6 tức là phiên bản cũ hơn vẫn gõ tv được hr bạn?

Không, mình bắt đầu dùng từ bản 1.6 bạn ạ. Bản đó mình ko gõ được trong PTS.

**Comment by tuyenvm** [comment]
Mình sẽ xem lại phần AutoUpdate.
Không biết Bảng điều khiển bạn cấu hình thế nào nhỉ bạn chụp hình gửi lên đây giúp mình được không.
Mình mới thử lại với Photoshop CC 2018 thì vẫn bình thường.

**Comment by thuyld** [comment]
> Mình sẽ xem lại phần AutoUpdate.
> Không biết Bảng điều khiển bạn cấu hình thế nào nhỉ bạn chụp hình gửi lên đây giúp mình được không.
> Mình mới thử lại với Photoshop CC 2018 thì vẫn bình thường.

bảng điều khiển mình để mặc định, chứ cũng ko chỉnh gì cả. chỉ uncheck cái spelling & Gramma thôi

**Comment by tuyenvm** [comment]
bạn đang dùng macOS bản nào nhỉ để mình check lại

**Comment by thuyld** [comment]
> bạn đang dùng macOS bản nào nhỉ để mình check lại

Mình dùng bản MacOS 10.12.3

**Comment by tuyenvm** [comment]
@thuyld bạn cập nhật bản 1.6.5 xem còn bị tình trạng này không nha bạn.

**Comment by thuyld** [comment]
> @thuyld bạn cập nhật bản 1.6.5 xem còn bị tình trạng này không nha bạn.

Vẫn bị bạn nhé. Thêm nữa là upgrade lên thì mất hết cả file openkey trong Applications luôn. 
Với mình ko cài được file .dmg mà phải xài file zip
![Screen Shot 2019-10-02 at 15 04 08](https://user-images.githubusercontent.com/55678638/66027964-40314700-e526-11e9-897b-6dd834e273e3.png)

**Comment by tuyenvm** [comment]
À mình hiểu nguyên nhân rồi, chủ yếu do file dmg không mở được nhưng trình AutoUpdate vẫn tự xóa file OpenKey.app trong  Application, tạm thời bạn tắt tính năng tự update đi nhé.
Mình sẽ cài lại macOS 10.12 để kiểm tra.
Thanks bạn
